main.go

```diff
@@ -44,18 +44,18 @@ func main() {
 			continue
 		}
 
-		log.Printf("⏬ %s", k)
+		logger.Successf("⏬ %s", k)
```

Comment:
This message isn't indicating any success.
```suggestion
		logger.Infof("⏬ %s", k)
```

Reply:
Perfect! Thank you! :smile: